Dockerfile

```diff
@@ -1,3 +1,20 @@
+FROM gradle:7-jdk11-alpine AS build
+
+COPY --chown=gradle:gradle . /build
+WORKDIR /build
+
+ARG VERSION_TAG
+RUN ./gradlew shadowJar \
+    --no-daemon \
+    -Prelease.forceVersion=$VERSION_TAG
+
+
 FROM openjdk:11
-COPY main/build/libs/*.jar /
+COPY --from=build /build/main/build/libs/*.jar /
```

Comment:
Per my previous comment, you are still pulling in multiple jars with this, only one of which is needed.  I think you can simplify this a lot by going with the following pattern:

```
COPY --from=build /build/main/build/libs/gtfs-validator-*-cli.jar /gtfs-validator-cli.jar
...
ENTRYPOINT [ "java", "-jar", "gtfs-validator-cli.jar" ]
```

Specifically, you won't need to include the version tag in the output jar with this approach.

Reply:
I dig it!